### DISCUSSION TITLE
Does the library support drawings (shapes, drawing tools, etc)?

### DISCUSSION THREAD

**Discussion by kevinsherman** [tradingview/lightweight-charts#661] [Q&A] [answered]
I'd like to be able to draw a trend line on a chart and then get it's value programmatically. is that possible?

**Comment by timocov** [comment, marked as answer]
The library does not support drawings in any kind at the moment, and we do not have any plan to add them to the library, but some or drawings (like trend line) might be emulated via line series with 2 points - the start and the end of trend line. You can see #116 for examples of such trend lines.
